packages/faceted-search/src/components/AdvancedSearch/AdvancedSearch.component.js

```diff
@@ -10,6 +10,7 @@ import { getTheme } from '@talend/react-components/lib/theme';
 import { useFacetedSearchContext } from '../context/facetedSearch.context';
 
 import theme from './AdvancedSearch.scss';
+import { PENDO_TAGS } from '../../constants';
```

Comment:
We should avoid using "PENDO" everywhere, this repo is public and we don't need to link Pendo to our repo at all.
USAGE_TRACKING_TAGS should be enough generic for example

Reply:
Totally agree, good point. Just made the change